docs/tables/crowdstrike_host.md

```diff
@@ -0,0 +1,48 @@
+# Table: crowdstrike_host
+
+Hosts are endpoints that run the Falcon sensor.
+
+## Examples
```

Comment:
There are a lot of columns in the `crowdstrike_host` table, can you please add more examples exploring some of these? For instance, what's inside the `policies` column? Can we show how to query that successfully? Also, can we add any examples around agent information, host version, etc.